### PR TITLE
Added simple Ollama example

### PR DESCRIPTION
**Pull request by akhilbandhu**
## 📥 Pull Request

**📘 Description**
Added an ollama example to satisfy Issue https://github.com/AgentOps-AI/agentops/issues/460

**🧪 Testing**
Tested out changes in local UI

### CODE REVIEW

File: examples/ollama_examples/ollama_examples.ipynb
Comment (areibman) [inline examples/ollama_examples/ollama_examples.ipynb:1]: image

Either only run `agentops.init()` without `agentops.start_session()` or run `agentops.init(auto_start_session=False)` plus `agentops.start_session()`

Comment (areibman) [inline examples/ollama_examples/ollama_examples.ipynb:1]: image
2 things:
1. Restart the kernel and run from top to bottom to ensure it works. Here, the first cell is `[14]` which may indicate the notebook state is wrong
2. Can you clear the outputs from the install cell before committing the save? Makes the notebook easier to read

**Review by areibman — CHANGES_REQUESTED** [review]
Very minor changes, but overall looks great!

File: docs/v1/examples/examples.mdx
Comment (areibman) [inline docs/v1/examples/examples.mdx:1]: Can you create a custom examples page? You can look at the LangChain example as a start.

You probably wouldn't know this, but we have a tool that dynamically injects the outputs of the example notebook into the page. i.e.
image

**Review by areibman — CHANGES_REQUESTED** [review]

File: docs/v1/examples/examples.mdx
Comment (akhilbandhu) [reply docs/v1/examples/examples.mdx:1]: @areibman Sure I can do that, what is the name of the tool you mentioned? Didn't want to go around replicating work. Let me know what it is, and I can get that pushed up in the next commit

Comment (areibman) [reply docs/v1/examples/examples.mdx:1]: Including the line `{/*    SOURCE_FILE: ...` should do the trick. You can copy the LangChain example .mdx file and use that as the starting point

Comment (akhilbandhu) [reply docs/v1/examples/examples.mdx:1]: Cool beans, I am assuming that that will only run when this is merged with main? Looks like it's a Github action

Comment (akhilbandhu) [reply docs/v1/examples/examples.mdx:1]: Actually was able to figure out the nbconvert tool and got it working! @areibman Let me know what you think

**Review by areibman — APPROVED** [review]
Awesome!!

**Pull request merged**